README.md

```diff
@@ -54,12 +54,12 @@ The wheels are then in the `target/wheels` directory.
 
 ## Getting embeddings
 
-rust2vec uses its own embedding format, which supports memory mapping,
+finalfusion uses its own embedding format, which supports memory mapping,
 subword units, and quantized matrices. GloVe and word2vec embeddings
-can be converted using rust2vec's `r2v-convert` utility.
+can be converted using finalfusion's `ff-convert` utility.
 
 Embeddings trained with
-[finalfrontier](https://github.com/danieldk/finalfrontier) version
+[finalfrontier](https://github.com/finalfusion/finalfrontier) version
```

Comment:
This can point to:

https://finalfusion.github.io/finalfrontier

Maybe this section should also have a link to:

https://finalfusion.github.io/pretrained